src/instructlab/lab.py

```diff
@@ -972,12 +912,12 @@ def convert(self, value, param, ctx) -> "torch.device":
 )
 @click.option(
     "--device",
-    type=TORCH_DEVICE,
+    type=click.Choice(["cpu", "cuda", "hpu"]),
     show_default=True,
     default="cpu",
     help=(
-        "PyTorch device for Linux training (default: 'cpu'). Use 'cuda' "
-        "for NVidia CUDA / AMD ROCm GPU, 'cuda:0' for first GPU."
+        "PyTorch device for Linux training. Use 'cuda' "
+        "for NVidia CUDA / AMD ROCm GPU, to use specific GPU, set visible GPU before run train command."
```

Comment:
"set visible GPU before", how should users do that?

Reply:
I think it's a little out of instructlab's scope. It's up to gpu type: NVidia CUDA / AMD ROCm GPU.  I suggest leave this to gpu user. The gpu user should know this well.
https://docs.nvidia.com/deploy/mps/#mps-on-multi-gpu-systems
https://rocm.docs.amd.com/en/latest/conceptual/gpu-isolation.html

Reply:
I think it's a little out of instructlab's scope. It's up to gpu type: NVidia CUDA / AMD ROCm GPU.  I suggest leave this to gpu user. The gpu user should know this well.
https://docs.nvidia.com/deploy/mps/#mps-on-multi-gpu-systems
https://rocm.docs.amd.com/en/latest/conceptual/gpu-isolation.html